static/js/components/forms/RegisterEmailForm.js

```diff
@@ -45,6 +46,15 @@ const RegisterEmailForm = ({ onSubmit }: Props) => (
             component={EmailInput}
           />
           <ErrorMessage name="email" component={FormError} />
+          <p className="py-2">
+            By creating an account I agree to the{" "}
+            <a href={routes.informationLinks.termsOfService}>
```

Comment:
Should the link not be opened in a new tab ?

Reply:
I thought the user doesn't need to go to a new tab he needs to read `Policy links` because they are going to be a decisive factor for the user at that point and we should keep the same tab for it and let them come back to register.

Additionally, I also checked the policy links with the footer and they also open in the same tab so the behavior would also be standardized if I keep this behavior.

Does this answer your concern @asadiqbal08 ?

FYI, @pdpinch